components/aws/athena/query/src/query.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import logging
+import time
+import re
+
+import boto3
+
+
+def get_client(region=None):
+  """Builds a client to the AWS Athena API."""
+  client = boto3.client('athena', region_name=region)
+  return client
+
+def query(client, query, database, output):
+  response = client.start_query_execution(
+      QueryString=query,
+      QueryExecutionContext={
+          'Database': database
+          },
+      ResultConfiguration={
+          'OutputLocation': output,
+          }
+      )
+
+  execution_id = response['QueryExecutionId']
+  logging.info('Execution ID: %s', execution_id)
+
+  # Athena query is aync call, we need to fetch results and wait for execution
+  state = 'RUNNING'
+  max_execution = 5 # TODO: this should be an optional parameter from users. or use timeout
```

Comment:
Yes - can be a default param in input

Reply:
Thanks. I will give to our users and see how do they like it.  Change will be made and expose to users later either with max_execution or max_waiting_time.